Dockerfile

```
@@ -22,224 +22,189 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-# Use Ubuntu 20.04 LTS
-FROM nipreps/miniconda:py38_1.4.2
+# Ubuntu 22.04 LTS - Jammy
+ARG BASE_IMAGE=ubuntu:jammy-20230308
 
-ENV DEBIAN_FRONTEND=noninteractive
+#
+# fMRIPrep wheel
+#
+FROM python:slim AS src
+RUN pip install build
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends git
+COPY . /src/fmriprep
```

DO NOT ACCEPT

@eilidhmacnicol @oesteban @mgxd One approach we could take to avoid annoying `.dockerignore` mismatches could be:

```suggestion
COPY .git /src/fmriprep/.git
RUN git -C /src/fmriprep reset --hard HEAD
```

This would ensure that images only get built from commits, and not from dirty worktrees. I'm not sure whether we value being able to build from a dirty worktree... Personally, I just `fmriprep-docker ... --patch fmriprep=$PWD/fmriprep` when I want that.